### DISCUSSION TITLE
web application not being indexed to search results.

### DISCUSSION THREAD

**Discussion by SkyBulk** [vercel/community#6576] [Help] [answered]
### Question

I built an application using nextjs deployed in Vercel; however, I notice that Google isn't indexing this site even when I have sitemap-0.xml, and robots. 

URL: https://slideshare2pdf.vercel.app/

![image](https://github.com/vercel/community/assets/43649816/01eac868-6a98-48c4-a4a8-c9b3cc05710f)
![image](https://github.com/vercel/community/assets/43649816/4d1e1792-7924-4057-b863-b6a8fad4846b)

**Comment by versecafe** [comment, marked as answer]
@SkyBulk indexing can take several weeks, and I would recommend you sign up and verify your site with the google search console and bings webmaster tools.
